netlify.toml

```
@@ -2,3 +2,4 @@
   base    = "website"
   publish = "website/build"
   command = "npm run build && cp _redirects ./build"
+  ignore  = "false"
```

```suggestion
  ignore  = "git diff --quiet HEAD^ HEAD -- docs/ website/"
```

This is what I mean we should use. This prevents code-only PRs from being built by Netlify, but also should resolve your issues on your fork.

---

I agree, this should fix the problem of new forks, too. I simply did not intend to change the current behavior of *this* site, but this should be an improvement. Let me check this on my fork first.

---

Git-diff runs on the base directory (`/website`), so the correct command was:

```
ignore = "git diff --quiet HEAD^ HEAD -- ../docs/ ."
```

Otherwise this approach seems to be working fine on my fork.

---

Ah, missed that. Thanks!